essay.tex

```diff
@@ -11,35 +11,71 @@
+\\~\\
+Some teams like to change up the sprint duration during development because they might be at a different stage that requires more or less time for sprint runs. So for example a team could start off developing a game with 3 week long sprints and then change to 2 week long sprints once they have finished the alpha stage of the game and can now work on implementing more content instead of working on the overall framework.
+
+\section{The idea of 1 week sprints?}
+1 week long sprints are rarely mentioned and are a fairly controversial idea, and for good reasons. This Essay wants to shed some light onto the positive and negatives of the weekly sprint and argue for why it's actually a good idea and that game dev teams should practice it more.
+\\~\\
+So let's start off with the negatives:\\
+Some sources say that 1 week sprints are too short and don't give enough time for teams to work everything they wanted to work on from their sprint planning, other sources say that 1 week sprints aren't good because it means a loss of time from having to do more sprint planning meetings and review meetings.
+\\~\\
+Now let's look at the positives:\\
+1 week or 5 working days is actually quite a long time when you think about it and a lot can be achieved during that time frame.
+An article by Dan Ackerson suggests that developers get sloppy and don't work as productive as productive as they could be in sprints that last more than a week and encourages the idea of 1 weeks sprints as it gets the team more focused on the sprint goals\cite{six}.
+Rob Galanakis in his article ``Two weeks is the worst sprint length`` says that he prefers 1 week long sprints as they allow for more feedback and chances to improve, but he admits that you need a confident team in order to have successful weekly sprints\cite{seven}.
+
+The weekly sprint idea doesn't get mentioned enough and really should, as it's a great way for experimenting different approaches to development cycles. But it's more of a preference at the end of the day, some teams might enjoy it more than others.
 
-\section{Conclusion}
 
+\section{Conclusion}
+\iffalse
 Write your conclusion here. The conclusion should do more than summarise the essay, making clear the contribution of the work and highlighting key points, limitations, and outstanding questions. It should not introduce any new content or information.
-
-\bibliographystyle{ieeetran}
+\fi
+In the end the is no real ideal sprint length, the length of a sprint entirely depends on what the team is capable of doing depending on the scope and scale of a game development project. The team decides, they get to choose what will best suite their needs for developing a game.
```

Comment:
This isn't a meaningful take away point. What do you recommend? Under what circumstances is a short or a long sprint preferable? By what kind of teams? Is a long sprint ever justified?